llm_assistant/ollama/openai_chat.py

```diff
@@ -0,0 +1,19 @@
+from openai import OpenAI
+import argparse
+client = OpenAI(base_url="http://0.0.0.0:8000", api_key="anything")
```

Comment:
Missing the code where we read in the `OPENAI_API_KEY` from the environment. Doing it like this will make everyone who wants to test the script modify this file.

Reply:
We don’t add the OPENAI_API_KEY here since we already use the key in the proxy server. 
Despite this, we still need to include some values fro the api_key because the OpenAI SDK requires an api_key value for proper execution.
So this value can be anything.
Please see [here](https://github.com/BerriAI/litellm), section "Quick Start Proxy - CLI" for more information.
I will update the README and add some comments here in the code to prevent confusion.

Reply:
Then in this case, incase of a generic comment, can we reference why we're doing that in the code as well? Something like

```python
# We don't need an actual api_key here. See `ollama/README.md`
client = OpenAI(base_url="http://0.0.0.0:8000", api_key="FAKE")
```